edx_proctoring/urls.py

```diff
@@ -30,6 +30,11 @@
         name='edx_proctoring.proctored_exam.attempt'
     ),
     url(
+        r'edx_proctoring/v1/proctored_exam/allowance/{}$'.format(settings.COURSE_ID_PATTERN),
+        views.ExamAllowanceView.as_view(),
+        name='edx_proctoring.proctored_exam.allowance'
+    ),
```

Comment:
I think the course_id should come earlier in the URL pattern:

edx_proctoring/v1/proctored_exam/{}/allowance

Also, we'll have to check the COURSE_ID_PATTERN regex, we do have a "new style" format. There is a regex defined in edx-platform. I'll have to dig it up.